### PR TITLE
filter_kgraph_std_dev

### PR DESCRIPTION
**Pull request by finnagin**

### CODE REVIEW

**Comment by finnagin** [comment]
FYI: input/output json still needs to be made and documentation generated etc. 

Currently this is just a rough draft for the yaml file.

**Comment by dkoslicki** [comment]
@finnagin is this ready for review?

**Comment by finnagin** [comment]
@dkoslicki I believe this, PR #31 , and PR #32 are all ready for review.

File: operations/filter_kgraph_std_dev.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      type: bool
+      example: true
+      default: false
+    top:
+      description: Indicate whether or not the threshold should be placed in the top or bottom of the values. E.g. top set as True will set the cutoff for filtering as the mean + threshold * std_dev while setting top to False will set the cutoff as the mean - std_dev * threshold.
+      type: bool
+      example: false
+      default: true
+    edge_keys:
+      description: This indicates if you only want to filter on specific edge_keys. If not provided or empty all edges will be filtered on and edge_key will not be considered when filtering.
```
Comment (patrickkwang) [inline operations/filter_kgraph_std_dev.yml:45]:
```suggestion
      description: This indicates if you only want to filter on specific edge_keys. If not provided or empty, all edges will be filtered on.
```

File: operations/filter_kgraph_std_dev.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      type: array
+      example: [e01]
+      default: null
+    qnode_keys:
+      description: This indicates if you only want nodes corresponding to a specific list of qnode_keys to be removed. If not provided oe empty no nodes will be removed when filtering.
```
Comment (patrickkwang) [inline operations/filter_kgraph_std_dev.yml:50]:
```suggestion
      description: This indicates if you only want nodes corresponding to a specific list of qnode_keys to be removed. If not provided or empty, no nodes will be removed when filtering.
```

**Review by patrickkwang — CHANGES_REQUESTED** [review]
Looks good! Minor suggestions.

**Review by dkoslicki — APPROVED** [review]

**Review by patrickkwang — APPROVED** [review]

**Pull request merged**